### ISSUE TITLE
Have consistent image resolution & format, provide alt text

### ISSUE THREAD

**Issue by TomasBeuzen** [UBC-MDS/py-pkgs#29]
- Images should be of consistent (high) resolution throughout the text.
- Images should all be .png
- Every image should have alt text specified ([we can do this easily with MyST](https://jupyterbook.org/content/figures.html#figure-parameters))
